src/components/SendLinkModal.jsx

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  ChakraProvider,
+  Text,
+  Input,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+} from '@chakra-ui/react';
+import Checkmark from '../assets/Check.png';
+import { sendInviteLink, NPOBackend } from '../utils/auth_utils';
```

Comment:
🚫 **[eslint]** <[import/named](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/named.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>sendInviteLink not found in '../utils/auth_utils'